src/Renderer/ThreeExtended/GlobeControls.js

```diff
@@ -685,17 +717,49 @@ function GlobeControls(camera, target, domElement, engine) {
         offset.copy(this.camera.position);
         offset.applyMatrix4(cameraTargetOnGlobe.matrixWorldInverse);
         spherical.setFromVector3(offset);
+
+        if (state === CONTROL_STATE.ORBIT && (Math.abs(snapShotSpherical.phi - spherical.phi) > delta || Math.abs(snapShotSpherical.theta - spherical.theta) > delta)) {
+            this.dispatchEvent({
+                type: 'orientationchanged',
+                old: {
+                    tilt: snapShotSpherical.phi * 180 / Math.PI,
+                    heading: snapShotSpherical.theta * 180 / Math.PI,
+                },
+                new: {
+                    tilt: spherical.phi * 180 / Math.PI,
+                    heading: spherical.theta * 180 / Math.PI,
+                },
+            });
+        } else if (state === CONTROL_STATE.PAN) {
+            this.dispatchEvent({
+                type: 'panchanged',
+            });
+        }
+
+        const oldRange = snapShotSpherical.radius;
+        const newRange = this.getRange();
+        if (Math.abs(newRange - oldRange) / oldRange > 0.001) {
+            this.dispatchEvent({
+                type: 'rangechanged',
+                old: { range: oldRange },
+                new: { range: newRange },
+            });
+        }
+
         state = CONTROL_STATE.NONE;
         lastRotation = [];
         if (enableTargetHelper) {
             this.dispatchEvent(this.changeEvent);
         }
 
-        this.dispatchEvent({
-            type: 'camera-target-updated',
-            oldCameraTargetPosition: oldCoord,
-            newCameraTargetPosition: C.fromXYZ(scene.referenceCrs, cameraTargetOnGlobe.position),
-        });
+        if (cameraTargetOnGlobe.position.distanceTo(oldCameraTargetOnGlobe) / spherical.radius > delta) {
+            this.dispatchEvent({
+                type: 'camera-target-updated',
+                old: { cameraTargetPosition: C.fromXYZ(scene.referenceCrs, oldCameraTargetOnGlobe) },
```

Comment:
There should be some coherency between the event name and the attribute content, to ease API usage.
If the event is `camera-target-updated` (I think I'd prefer `-changed` btw), the expected attribute would be: `event.new.cameraTarget`